filament/include/filament/RenderableManager.h

```diff
@@ -488,6 +488,9 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
     void setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
             MorphTargetBuffer* morphTargetBuffer, size_t count); //!< \overload
 
+    void setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
```

Comment:
This needs to be added to the Java APIs as well.

Reply:
Yes I will create a new PR for these changes.